src/odemis/driver/test/tescan_test.py

```diff
@@ -189,6 +372,7 @@ def test_probe_current(self):
         time.sleep(6)  # Wait for value refresh
         self.assertAlmostEqual(orig_probe_current, ebeam.probeCurrent.value)
 
+    # @skip
```

Comment:
Is this line necessary? If so, please add a comment line to explain why it is there and commented out.

Reply:
It was a workaround for running it with the Amber simulator, since this class was written for the Mira simulator. With the Mira simulator running it actually just works fine. For now I will add a comment to it as you suggested!